osmomath/binary_search.go

```diff
@@ -144,7 +144,7 @@ func (e ErrTolerance) CompareBigDec(expected BigDec, actual BigDec) int {
 // Binary search inputs between [lowerbound, upperbound] to a monotonic increasing function f.
 // We stop once f(found_input) meets the ErrTolerance constraints.
 // If we perform more than maxIterations (or equivalently lowerbound = upperbound), we return an error.
-func BinarySearch(f func(input sdk.Int) (sdk.Int, error),
+func BinarySearch(f func(sdk.Int) (sdk.Int, error),
```

Comment:
I don't think this makes a difference because https://github.com/osmosis-labs/osmosis/pull/3904 tags `main` directly and passes CI